safe_transaction_service/tokens/tests/test_models.py

```diff
@@ -126,3 +126,24 @@ def test_fix_balancer_pool_tokens(self, get_metadata_mock: MagicMock):
         self.assertEqual(
             token.name, "Balancer Pool Token " + get_metadata_mock.return_value.name
         )
+
+    from gnosis.eth.ethereum_client import Erc20Info, Erc20Manager
```

Comment:
Why do you do the import here?

Reply:
My bad. Editor's view and I thought that with the commit hooks it'd put them in the correct place 😬

Reply:
919bfdc